bot/core/bot.py

```diff
@@ -47,4 +88,13 @@ async def on_ready(self) -> None:
         print("Bot is ready")
 
     async def close(self) -> None:
+        """Close the bot and do some cleanup."""
         await super().close()
+        await self.pool.close()
+
+    @tasks.loop(hours=3)
+    async def change_status(self) -> None:
+        """Change the bot status every 3 hours."""
+        await self.change_presence(
+            activity=Game(name=next(self.status)),
+        )
```

Comment:
This should be removed due to the reasons stated in a comment above.

Reply:
Yep